components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -23,27 +23,27 @@ void Nudging::set_grids(const std::shared_ptr<const GridsManager> grids_manager)
   m_num_levs = m_grid->get_num_vertical_levels();  // Number of levels per column
 
   FieldLayout scalar3d_layout_mid { {COL,LEV}, {m_num_cols, m_num_levs} };
-  // Layout for horiz_wind field
-  FieldLayout horiz_wind_layout { {COL,CMP,LEV}, {m_num_cols,2,m_num_levs} };
 
   constexpr int ps = 1;
   auto Q = kg/kg;
   Q.set_string("kg/kg");
   add_field<Required>("p_mid", scalar3d_layout_mid, Pa, grid_name, ps);
   add_field<Updated>("T_mid", scalar3d_layout_mid, K, grid_name, ps);
   add_field<Updated>("qv", scalar3d_layout_mid, Q, grid_name, "tracers", ps);
-  add_field<Updated>("horiz_winds",   horiz_wind_layout,   m/s,     grid_name, ps);
-  
+  add_field<Updated>("u", scalar3d_layout_mid, m/s, grid_name, ps);  
+  add_field<Updated>("v", scalar3d_layout_mid, m/s, grid_name, ps);  
+
   //Now need to read in the file
   scorpio::register_file(datafile,scorpio::Read);
   m_num_src_levs = scorpio::get_dimlen_c2f(datafile.c_str(),"lev");
   double time_value_1= scorpio::read_time_at_index_c2f(datafile.c_str(),1);
   double time_value_2= scorpio::read_time_at_index_c2f(datafile.c_str(),2);
-
+    
   //Here we are assuming that the time in the netcdf file is in days
   //Internally we want this in seconds so need to convert
   //Only consider integer time steps in seconds to resolve any roundoff error
   time_step_file=int((time_value_2-time_value_1)*86400);
+  std::cout<<"time_step_file: "<<time_step_file<<std::endl;
```

Comment:
Avoid printing stuff to cout, and use the `log` method instead. If instead this was just some print statement during debugging, you can just prune it.

Reply:
Yes, I just noticed that I thought I had committed the removal of these debug statements but they are only in my local repo. This has been removed in the latest commit.